install.sh

```diff
@@ -105,13 +105,15 @@ if lspci | grep -E -i '(vga|3d|display)' | grep -i nvidia > /dev/null; then
 # AMD
 elif lspci | grep -E -i '(vga|3d|display)' | grep -i amd > /dev/null; then
 	if lspci -nnk | grep -i vga -A3 | grep 'Kernel modules: radeon, amdgpu' > /dev/null; then
```

Comment:
not sure why `amdgpu` is listed here as well, is this right?

Reply:
Yes, it's a new kernel module that replaced radeon kernel module some time ago.

Reply:
yes, but I mean, why is amdgpu string included when detecting radeon kernel module. Seems confusing. I thought we were detecting for amdgpu here until you added the new condition.

Reply:
There are 3 types of AMD Graphics cards of Linux :
1) Legacy / ATI (They use radeon kernel module)
2) Sea Island and Southern Island (They use radeon kernel module by default but its recommended to switch to amdgpu for better performance in Proton / Native games)
3) Modern GPUs - Polaris, Vega, Raven Ridge (They use amdgpu kernel module by default)

This detection is detecting all 3 types of those cards, we don't want to confuse users with wrong information